specification/resources/resource-manager/readme.md

```diff
@@ -40,35 +40,22 @@ tag: package-locks-2016-09
 tag: package-policy-2019-06
 ```
 
-``` yaml $(package-policy)
-tag: package-policy-2019-01
-```
-
 ``` yaml $(package-resources)
 tag: package-resources-2019-08
 ```
 
 ``` yaml $(package-subscriptions)
-tag: package-subscriptions-2018-06
+tag: package-subscriptions-2019-06
```

Comment:
what's the sorting order? it's a mess now.

Reply:
what do you mean, there's only one tag in here?